### PR TITLE
test: simplified editing of version

### PR DESCRIPTION
**Pull request by iyerashwin14**

### CODE REVIEW

File: src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ViewVersionSteps.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ */
+public class ViewVersionSteps extends Base {
+    /**
+     * Get user to the View Version Page.
+     * @param productVersionNumber is used to navigate to the view version page of that product
+     **/
+
+    @Given("^I navigate to edit version page of listing \"(.*)\"$")
+    public void editVersionPage(final String productVersionNumber) {
+        getDriver().get(getUrl() + "#/organizations/developers/1816/products/1742/versions/" + productVersionNumber);
```
Comment (andlar) [inline src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ViewVersionSteps.java]: These IDs should be passed in as variables

File: src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ViewVersionSteps.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    /**
+     * Edit the version of a product.
+     * @param productVersionNumber is the product version number passed in page objects
+     **/
+    @And("^I edit the version of \"(.*)\"$")
+    public void editVersion(final String productVersionNumber) {
+        ViewVersionPage.editVersionLink(getDriver(), productVersionNumber).click();
+        ViewVersionPage.editVersionField(getDriver()).clear();
+        ViewVersionPage.editVersionField(getDriver()).sendKeys(getCurrentDate());
```
Comment (andlar) [inline src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ViewVersionSteps.java:36]: I don't really like the use of getCurrentDate here, and I have some ideas as to what I'd rather have, but I think it should work, so am not going to require changes of this part.

**Review by andlar — CHANGES_REQUESTED** [review]

File: src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ViewVersionSteps.java (hunk 1 above)
Comment (ashwinimore) [reply src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ViewVersionSteps.java]: Agree with parametrizing IDs

File: src/test/resources/Features/OCD-2806-SimplifiedEditingOfVersion.feature [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    When I navigate to edit version page of developer "1816" and product "1742" of listing "5085"
+    Then I see merge link to merge the version of "5085"
+    Examples:
+      |ROLE       |
+      |ROLE_ADMIN |
+      |ROLE_ONC   | 
+
+Scenario Outline: User logs in as ROLE and edits a version
+    Given I'm logged in as "<ROLE>"
+    When I navigate to edit version page of developer "1816" and product "1742" of listing "<Product Version Number>"
```
Comment (andlar) [inline src/test/resources/Features/OCD-2806-SimplifiedEditingOfVersion.feature]: Can we rearrange this so it reads more like English?
```suggestion
    When I navigate to the version edit page for version "<version id>" of product "<product id>" of developer "<developer id>"
```

**Review by andlar — APPROVED** [review]

**Review by ashwinimore — APPROVED** [review]

**Pull request merged**